bitnami/influxdb/templates/influxdb/standalone-dpl.yaml

```diff
@@ -0,0 +1,253 @@
+            {{- end }}
+            {{- end }}
+            {{- if .Values.user.name }}
+            - name: INFLUXDB_USER
+              value: {{ .Values.user.name | quote }}
+            {{- if .Values.user.usePasswordFile }}
+            - name: INFLUXDB_USER_PASSWORD_FILE
+              value: "/opt/bitnami/influxdb/secrets/user-password"
+            {{- else }}
+            - name: INFLUXDB_USER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "influxdb.secretName" . }}
+                  key: user-password
+            {{- end }}
+            {{- end }}
+            {{- if .Values.readUser.name }}
+            - name: INFLUXDB_READ_USER
+              value: {{ .Values.readUser.name | quote }}
+            {{- if .Values.readUser.usePasswordFile }}
+            - name: INFLUXDB_READ_USER_PASSWORD_FILE
+              value: "/opt/bitnami/influxdb/secrets/read-user-password"
+            {{- else }}
+            - name: INFLUXDB_READ_USER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "influxdb.secretName" . }}
+                  key: read-user-password
+            {{- end }}
+            {{- end }}
+            {{- if .Values.writeUser.name }}
+            - name: INFLUXDB_WRITE_USER
+              value: {{ .Values.writeUser.name | quote }}
+            {{- if .Values.writeUser.usePasswordFile }}
+            - name: INFLUXDB_WRITE_USER_PASSWORD_FILE
+              value: "/opt/bitnami/influxdb/secrets/write-user-password"
+            {{- else }}
+            - name: INFLUXDB_WRITE_USER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "influxdb.secretName" . }}
+                  key: write-user-password
+            {{- end }}
+            {{- end }}
+            {{- if .Values.database }}
+            - name: INFLUXDB_DB
+              value: {{ .Values.database | quote }}
+            {{- end }}
+            {{- if .Values.influxdb.extraEnvVars }}
+            {{- tpl (toYaml .Values.influxdb.extraEnvVars) $ | nindent 12 }}
+            {{- end }}
+          ports:
+            - name: http
+              containerPort: 8086
+              protocol: TCP
+            - name: rcp
+              containerPort: 8088
+              protocol: TCP
+          {{- if .Values.influxdb.livenessProbe.enabled }}
+          livenessProbe:
+            exec:
+              command:
+                - bash
+                - -c
+                - |
+                  if [[ -f "${INFLUXDB_ADMIN_USER_PASSWORD_FILE:-}" ]]; then
+                      export INFLUXDB_ADMIN_USER_PASSWORD="$(< "${INFLUXDB_ADMIN_USER_PASSWORD_FILE}")"
+                  fi
+                  {{ if .Values.authEnabled }}INFLUX_USERNAME="$INFLUXDB_ADMIN_USER" INFLUX_PASSWORD="$INFLUXDB_ADMIN_USER_PASSWORD"{{ end }} influx -host 127.0.0.1 -port 8086 -execute "SHOW DATABASES"
```

Comment:
Just thinking out loud: does it make sense to make the probes commands also customizable in the `values.yaml`?

Reply:
We can customize everything in the yaml, even if we use "exec" or "httpGet" probes but the question is: Are we adding **real** value in it? I don't think so